solr/solr-ref-guide/modules/configuration-guide/pages/cluster-plugins.adoc

```diff
@@ -278,3 +276,34 @@ curl -X POST -H 'Content-type: application/json' -d '{
     }'
   http://localhost:8983/api/cluster/plugin
 ----
+
+== Plugin Configuration in solr.xml
+
+Plugins can also be installed by creating `<plugin>` elements in `solr.xml`.
+This might be preferred in some situations, such as enforcing immutable infrastructure.
+
+TIP: See also the section xref:configuration-guide:configuring-solr-xml.adoc[] for more information about the `solr.xml` file, where to find it, and how to edit it.
```

Comment:
I was unsure, because I had the impression that the trend was to split the documentation about `solr.xml` in more specialised pages. For example, metric configuration is already described on its own page.